Source/cs505/group1/state/ButtonAContext.java

```diff
@@ -0,0 +1,48 @@
+package cs505.group1.state;
+
+/**
+ * An example concrete subclass of ButtonContext.  
+ * 
+ * @author CS505 Fall17 Group1: <br>
+ * Emily Park<br>
+ * Jeffrey Blankenship  <br>
+ * Cecelia Oluwadoyinsola<br>
+ * James Luczynski<br>
+ * Melissa Mulcahy <br>
+ * @version 2017.10.29
+ */
+public class ButtonAContext extends ButtonContext{
+    // buttonState is the current state 
+    private ButtonState buttonState;

```

Comment:
This instance variable already exists on the parent abstract class so shouldn't be here on child class